gematria/granite/python/token_graph_builder_model.py

```diff
@@ -50,6 +52,20 @@ class TokenGraphBuilderModel(graph_builder_model_base.GraphBuilderModelBase):
   READOUT_VARIABLES = 'TokenGraphBuilderModel.readout'
   TASK_READOUT_VARIABLES = 'TokenGraphBuilderModel.task_readout'
 
+  INSTRUCTION_ANNOTATIONS_TENSOR_NAME = (
+      'TokenGraphBuilderModel.instruction_annotations'
+  )
+  ANNOTATION_NAMES_TENSOR_NAME = 'TokenGraphBuilderModel.annotation_names'
+
+  # A 1D byte tensor that contains the list of annotation names in the order of
+  # their indices in the graph builder.
+  _annotation_name_tensor: tf.Tensor
+
+  # The list of annotation names, in the order of their indices in the model.
+  _annotation_name_list: Sequence[str]
+
+  _instruction_annotations: tf.Tensor
+
```

Comment:
Since this is already in the base graph builder model, we should probably move all these definitions to that class (and make the annotations available in any graph builder model).

Eventually (but definitely not in this PR chain), we can probably merge `GraphBuilderModelBase` and `TokenGraphBuilderModel` because there is no other graph builder model anyway.